client/blocks/inline-help/popover.jsx

```diff
@@ -169,10 +183,17 @@ class InlineHelpPopover extends Component {
 			`inline-help__${ this.state.activeSecondaryView }`
 		);
 		return (
-			<div className={ classes }>
+			<section ref={ this.secondaryViewRef } className={ classes }>
 				{
 					{
-						[ VIEW_CONTACT ]: <InlineHelpContactView />,
+						[ VIEW_CONTACT ]: (
+							<Fragment>
+								<h2 className="inline-help__title" tabIndex="-1">
+									{ __( 'Contact Us' ) }
+								</h2>
+								<InlineHelpContactView />
```

Comment:
The code doesn't seem so straight forward and it seems that we add some hidden html just for accessibility. Would you give it a try, setting the focus with a `useEffect` inside the `InlineHelpContactView` ?

Moreover, since `InlineHelpContactView` has two states :
1) for cases that can't live chat. Especially in that case `Contact Us` is misleading, as we just present forum articles to the user
2) for cases that support live chat. In that case we already have a `Contact Us` title that we conditionally display

I would recommend (for each case): 
1) adding a `useEffect` here
https://github.com/Automattic/wp-calypso/blob/97624ceee07786f09e6217e5a3454cdfa9552a88/client/blocks/inline-help/inline-help-forum-view.jsx#L23-L25

2) tweaking here
https://github.com/Automattic/wp-calypso/blob/aa852bec11256f57bb8e47ee6b3c73bba61d111d/client/me/help/help-contact/index.jsx#L618-L622

Reply:
I agree that adding the focus and reusing items already in place is a better idea. Originally I had simply focused the wrapping container, but iOS doesn't move focus to the container. The issue with moving focus to these existing titles is that they are not immediately loaded, and we need to move focus immediately (ideally). 

Sometimes the form can take awhile to load, in which case:
-  the button that was focused is already gone,
- so focus gets reset to the top of the dom. 
- Then after form loads, focus would be sent to the popover, which would be a confusing experience.

Two ideas:
- Instead of "Contact Us," use a more generic name that fits both scenarios, like "Get Help"
- Leave "Contact Us," as the button they selected says "Contact Us" and doesn't seem to have any variation.

Reply:
Using 'Get Support` for now, seems ok!